### DISCUSSION TITLE
Vite 5 Support?

### DISCUSSION THREAD

**Discussion by develohpanda** [vikejs/vike#1310] [Discussion] [answered]
Hi there! Is Vite 5 supported in the current release ([0.4.147](https://github.com/vikejs/vike/compare/v0.4.146...v0.4.147))? Upgrading to Vite 5 is not something I have embarked on yet, and am currently trying to add Vike into my (fairly large) project.

**Comment by brillout** [comment, marked as answer]
Yes, Vike supports Vite 5. Vike actually always supports the latest Vite version as Vike is part of Vite's ecosystem CI.

Btw. is @Capgemini open to sponsor Open Source projects such as Vike?

**Reply by develohpanda** [reply]
TIL about [vite-ecosystem-ci](https://github.com/vitejs/vite-ecosystem-ci/blob/main/tests/vike.ts). That's neat, thank you!

> Btw. is https://github.com/Capgemini open to sponsor Open Source projects such as Vike?

I will definitely suggest it, and do what I can to push for it.

**Reply by brillout** [reply]
It really is neat :) Thanks!
